base/libgit2/libgit2.jl

```diff
@@ -490,6 +490,22 @@ set by `mode`:
   2. `Consts.RESET_MIXED` - default, move HEAD to `id` and reset the index to `id`.
   3. `Consts.RESET_HARD` - move HEAD to `id`, reset the index to `id`, and discard all working changes.
 
+# Example
+```julia
+# fetch changes
+LibGit2.fetch(repo)
+isfile(joinpath(repo_path, our_file)) # will be false
+
+# fastforward merge the changes
+LibGit2.merge!(repo, fastforward=true)
```

Comment:
I wonder if a more obvious method is available, because this is kind of bizarre.